include/graph/graph.h

```diff
@@ -0,0 +1,270 @@
+#ifndef GRAPH
+#define GRAPH
+
+#include "graph_descriptor.h"
+#include <any>
+#include <unordered_map>
+#include <unordered_set>
+#include <stack>
+#include<iostream>
+
+namespace FerryDB {
+	template<class Vertex, typename WeightType = int>
+	// Might not need Printable here, but added it in case we do need it
+		requires graph::PrintableGraphTypes<Vertex, WeightType> && graph::Hashable<Vertex>&& graph::Hashable<WeightType>
```

Comment:
Instead of making it Hashable, we could also create an integer ID and store is as the hash instead